src/SSW.SophieBot/components/SSWSophieBot.HttpClientComponents.PersonQuery/EmployeesHelper.cs

```diff
@@ -131,29 +197,41 @@ static int GetBusinessDays(DateTime start, DateTime end)
             }
         }
 
-        public static GetAppointmentModel GetAppointmentBy(DateTime date, List<GetAppointmentModel> appointments)
+        public static List<string> GetClientsByDate(DateTime date, List<GetAppointmentModel> appointments)
         {
-            var results = appointments
-                .Where(appointment => date.Ticks >= GetTicksFrom(appointment.Start) && date.Ticks <= GetTicksFrom(appointment.End))
-                .Where(appointment => !leavePhrases.Any(appointment.Subject.ToLower().Contains))
-                .ToList();
-            return results.Count != 0 ? results[0] : null;
+            var clientAppointments = GetEnumerableAppointmentsByDate(appointments, date)
+                .Where(appointment => !_internalCompanyNames.Contains(appointment.Regarding.ToLower()))
```

Comment:
`appointment.Regarding` could be `null`.We should do `appointment.Regarding?.ToLower()`, same to others.